action.yml

```diff
@@ -26,11 +27,34 @@ runs:
     env:
       INPUT_APP: ${{ inputs.app_name }}
       INPUT_TUTOR_VERSION: ${{ inputs.tutor_version }}
+  
+  - name: Prepare Tutor Nightly and Launch
+    if: ${{ inputs.tutor_version == 'nightly' }}
+    run: |
+      pip install virtualenv
+      virtualenv venv
```

Comment:
Why do we create a virtual environment?

Reply:
The reason why we need a virtual environment is because nightly can not be run in the root (user permissions) as you can see here https://github.com/eduNEXT/eox-tenant/actions/runs/10826637377/job/30040673637